lib/document/validation/validateDocumentFactory.js

```diff
@@ -68,12 +67,6 @@ module.exports = function validateDocumentFactory(
       return result;
```

Comment:
The next line already returns the result. We don't need this `if (!result.isValid()) {`